draft-ietf-httpbis-connect-tcp.md

```diff
@@ -44,9 +44,9 @@ HTTP/3 uses a UDP transport, so it cannot be forwarded using the pre-existing CO
 
 ## Problems
 
-Classic HTTP CONNECT proxies are identified by an origin.  The proxy does not have a path of its own.  This prevents any origin from hosting multiple distinct proxy services.
+Classic HTTP CONNECT proxies are identified by a host, a port number, and optionally a security protocol.  The proxy does not have a path of its own.  This prevents any HTTP origin from hosting multiple distinct proxy services.
 
-Ordinarily, HTTP allows multiple origin hostnames to share a single server IP address and port number (i.e., virtual-hosting), by specifying the applicable hostname in the "Host" or ":authority" header field.  However, classic HTTP CONNECT proxies use these fields to indicate the CONNECT request's destination ({{?RFC9112, Section 3.2.3}}), leaving no way to determine the proxy's origin from the request.  As a result, classic HTTP CONNECT proxies cannot be deployed using virtual-hosting, nor can they apply the usual defenses against server port misdirection attacks (see {{Section 7.4 of ?RFC9110}}).
+Ordinarily, HTTP allows multiple origin hostnames to share a single server IP address and port number (i.e., virtual-hosting), by specifying the applicable hostname in the "Host" or ":authority" header field.  However, classic HTTP CONNECT proxies use these fields to indicate the CONNECT request's destination ({{?RFC9112, Section 3.2.3}}), leaving no way to distinguish multiple proxies on one server.  As a result, classic HTTP CONNECT proxies cannot be deployed using virtual-hosting, nor can they apply the usual defenses against server port misdirection attacks (see {{Section 7.4 of ?RFC9110}}).  The absence of an explicit origin for the proxy also creates ambiguity about the use of origin-scoped response header fields such as Alt-Svc {{?RFC7838}} and Strict-Transport-Security {{?RFC6797}}.
```

Comment:
Expanding on my comment above, how about something like:
```suggestion
An ordinary HTTP request involves a URI, which includes an origin as a property. Since clients specify origins in HTTP requests using the "Host" or ":authority" header field, servers can colocate multiple origins on a single server IP address and port number, a practice known as virtual hosting. However, the classic CONNECT method is an exception. It targets a proxy, and the hostname of the proxy is not included in the HTTP request. As a result, classic CONNECT cannot be deployed using virtual hosting, nor can it implement the usual defenses against server port misdirection attacks (see {{Section 7.4 of ?RFC9110}}). The absence of an explicit origin for the proxy also creates ambiguity about the use of origin-scoped response header fields such as Alt-Svc {{?RFC7838}} and Strict-Transport-Security {{?RFC6797}}.
```